libs/newsletter/guides/Advanced/customizing-submission-data.md

```diff
@@ -0,0 +1,60 @@
+# Customizing Submission Data
+Whenever you are setting up your newsletter subscription within your application, you must decide what information is important for your user to include for the subscription. This guide will walk you through how to include all the desired information when the user submits their subscription using the `DaffNewsletter` library.
+
+## Table of Contents
+- [The DaffNewsletterSubmission](#the-daff-newsletter-submission)
+- [Creating your own submission object](#creating-your-own-submission-object)
+- [Utilizing your payload](#utilizing-your-payload)
+## The DaffNewsletterSubmission
+
+The `DaffNewsletterSubmission` is the object that holds your subscription data and gets sent to your application's backend. 
+```ts
+export interface DaffNewsletterSubmission {
+  email: string;
+}
+```
+It is simple and represents the baseline information needed in a subscription. 
+
+
+## Creating your own submission object
+
+To include all of the information you want in your newsletter subscription, you must define your own newsletter payload, which is going to represent all of the data required from the user when subscribing to your application's newsletter. To do so using the `DaffNewsletter`, you must use `DaffNewsletterSubmission`, or alternatively define a class that implements the `DaffNewsletterSubmission`. The `DaffNewsletterSubmission` only contains one variable of `email` to be used as a standard template for the Newsletter library, so it must be included in your defined class.
```

Comment:
> To do so using the `DaffNewsletter`, you must use `DaffNewsletterSubmission`, or alternatively define a class that implements the `DaffNewsletterSubmission`

To customize `@daffodil/newsletter` to use your custom payload when sending a submission, you must define an object that implements the `DaffNewsletterSubmission`.